meetings/2025-11-26.md

```diff
@@ -0,0 +1,184 @@
+* SC: The current chairs did not provide an explanation as to why they are not adhering to the charter.
+* SC: In order to carry out the election as per charter, current chairs must either:
+  * a. Abide to the charter and carry out the remaining tasks
+  * b. Step down from their role so that the CG can carry out the tasks with chairs that are willing. If no chairs, volunteers can carry out the election.
+  * c. Another option where a consensus can be reached in the CG if there are strong reasons as to why options a. and b. are not acceptable.
+* SC: First, the current chairs can response to the above. The CG participants (not limited to those present in this meeting) can then follow up with next steps.
+* VB: I subscribe to all of the above expressed by SC.
+* SC: So, remaining question is whether or not the obligations imposed by the charter will be followed, or what the situation is regarding the election.
+Do the chairs intent to follow the charter as it stands, or do we need to figure out a different way forward.
+* MdJ: Why is the list of participants needed - if the WBS is used for authenticating a CG member.
+* VB: Are you familiar with the discussion around this? There were multiple instances of this discussion online.
+* MdJ: Not familiar with the discussion.
+* SC: One is who is a CG member, the other is the snapshot characteristics (the point in time is not that important). So mainly, it is to cross check with the W3C staff such that we can use the WBS, where indeed the CG participants can voke, but within there there is an additional filter to rule out anybody who is not eligible. It is more of a procedural thing, rather than an all-hell-breaks-loose scenario. So, there was the convenience of having W3C accounts in the WBS system.
+* MdJ: What could go wrong? 
+* VB: https://github.com/w3c-cg/solid/issues/35#issuecomment-3570472974 
+* MdJ: So, in practise there is a small difference but indee this is a matter of process.
+* VB: interjects: It is also a form of safeguard - just because we can't prevent everything under the sun does not mean we should not take care of some common sense things. *(did i get that right?), thanks.)
+* eP: there was discussion online, arguments were made. I am not convinced, I want to propose ammenemnds. But I am fine with doing this election by the charter - if we find people who want to run the election. 
+* MdJ: Just checking, you propose to pause the election, do we also require re-bidding for co-chair?
+* eP: No, just delaying until we are confrom to charter, i.e., two weeks after publication of the snapshot list.
+* MdJ: I don't want to stand in the way for this group to re-run the election by the charter. For my taste, there was too much discussion around process rather than building things. So, I will step down after this meeting.
```

Comment:
```suggestion
* MdJ: I don't want to stand in the way of this group running the election by the charter. For my taste, there was too much discussion around process rather than building things. So, I will step down after this meeting.
```